companies/github.md

```diff
@@ -4,8 +4,8 @@ tags: ["tech"]
 blm_statements:
   - url: https://twitter.com/natfriedman/status/1266790816174882816?s=20
     date_posted: 2020-05-30
-    type: Tweet
-    raw_text: This week has been a horrifying, sad reminder of the centuries-long pattern of systemic racism in the US. And that our criminal justice system is in dire need of reform. GitHub stands with the Black community and will not be silent on violence and injustice.
+    type Tweet
```

Comment:
@Nyette I would expect this file to error but looks like it passed 😭

Reply:
We may have to play around with the [indentation](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.indentation) rule

Reply:
May I add a configuration file?

Reply:
yep definitely!